starwars/src/components/Character/SWCharacters.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import './StarWars.css';
+
+class Character extends Component {
+    constructor() {
+        super();
+        this.state = {
+
+        }
+
+    }
+
+    componentDidMount() {
+        this.setState({names: this.props.datum.name})
+    }
+
+    render() {
+        return (            
+            <div className="character">
+                <div className="thumbnail"></div>
+                <div>
+                <div className="character__description"><div className="prop">Name:</div>{this.props.datum.name}</div>
```

Comment:
Look into your div structure. It can probably be cleaned up a little. Let me know if you want some suggestions.

A personal preference of mine is not to add classes until I need to use them, even when I plan on using them later. This is because I may never end up using them because of my plan/app structure changing.